content/articles/creating-a-simple-chat-app-in-react-using-the-chat-engine-api/index.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Creating a Simple Chat App in React.js using the chat engine API
+---
+
+This article will teach the React.js way of creating a chat application using the chat engine API. It aims at making it less cumbersome compared to when one opts the Node.js and socket.io way instead.
+
+The learner will understand how the chat engine API works and how to use it to create a precise yet functional direct-messaging application.
+
+
+Approximating to other APIs that deliver user responses to the system, and sends back the system's responses to the user, this API works the same way, with better functionalities.
+
+This tool saves you a lot of implementation time otherwise needed on many lines of code.
+Chat engine API can implement; 
+ - A group chat application.
+ - Direct messages chat application that would work like Facebook messenger .
+
+### Prerequisites
+You will need to be equipped with what follows next before jumping right to it.
+1. Primary React.js library mastery. Head to [the official documentation](https://reactjs.org/tutorial/tutorial.html) if there's a need to familiarize yourself with React.js.
```

Comment:
Do not make this sentence long, simplify. For example, `1.[Node.js](link).` While you're informing reader of the knowledge required, you're at the same time giving them links to refer to.